docs/examples/progress/dashboard-progress-bar.vue

```diff
@@ -50,8 +50,8 @@ export default defineComponent({
       ...toRefs(state),
       increase,
       decrease,
-      Minus,
-      Plus,
+      Minus: shallowRef(Minus),
+      Plus: shallowRef(Plus),
```

Comment:
There are lots of files with the same issue, I'm not going to point them out one by one, please address them all so we can get it merged.